### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by piper511** [community/community#68568] [Programming Help]
### Select Topic Area

Question

### Body

How do you use this thing? Can I do Microsoft here and download things here?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @piper511 ! You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.

To answer your questions: Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

As for Microsoft, GitHub is a software hosting service, we host millions of projects, so it is likely that they have software that is also hosted here. However, they have their own community: https://answers.microsoft.com/en-us that will help answer your questions on them!
